Jellyfin.Server/Startup.cs

```diff
@@ -46,7 +52,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddCustomAuthentication();
 
             services.AddJellyfinApiAuthorization();
-            services.AddHttpClient();
+
+            services
+                .AddHttpClient(NamedClient.Default, c =>
+                {
+                    c.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(_applicationHost.Name.Replace(' ', '-'), _applicationHost.ApplicationVersionString));
```

Comment:
Minor thing, but maybe put `new ProductInfoHeaderValue(_applicationHost.Name.Replace(' ', '-'), _applicationHost.ApplicationVersionString)` in a variable.